### PR TITLE
fix: cli dbt 1.5 manifest

### PR DESCRIPTION
**Pull request by rephus**
Closes: #5667

### Description:

Screenshot from 2023-06-02 09-51-44
Screenshot from 2023-06-02 09-51-52

using manifest v7 on dbt 1.4.1

Screenshot from 2023-06-02 10-00-27

### CODE REVIEW

File: packages/cli/src/schemav9.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                            "$ref": "#/definitions/LightdashColumnMetadata"
+                        }
+                    }
+                }
+            ]
+        },
+        "LightdashCompiledModelNode": {
+            "allOf": [
+                {
+                    "$ref": "https://schemas.getdbt.com/dbt/manifest/v9.json#/definitions/ModelNode"
```
Comment (rephus) [inline packages/cli/src/schemav9.json:151]: CompiledModelNode from v7 is called ModelNode on v9

File: packages/cli/src/dbt/manifest.ts [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
+export const getDbtManifest = async (): Promise<DbtManifestVersion> => {
+    const version = await getDbtVersion();
+    if (version.startsWith('1.5.')) return DbtManifestVersion.V9;
+    return DbtManifestVersion.V7;
+};
+
+export const getManifestPath = async (targetDir: string): Promise<string> => {
+    const version = await getDbtVersion();
+    if (version.startsWith('1.5.'))
+        return path.join('./target', 'manifest.json');
```
Comment (rephus) [inline packages/cli/src/dbt/manifest.ts:24]: default target path has changed on 1.5

**Comment by rephus** [comment]
1.5 doesn't like this derived metric from the dbt semantic layer, I removed it, I guess it is going to be deprecated soon anyway.. https://github.com/lightdash/jaffle_shop/commit/5a5f95f74a9b3ea38620dcd54984581b07613d5f 

Screenshot from 2023-06-05 08-27-42

**Review by joaoviana — APPROVED** [review]
Looks great and worked locally 👍

File: packages/cli/src/handlers/dbt/compile.ts [hunk 3]
```diff
@@ -54,7 +54,8 @@ const optionsToArgs = (options: DbtCompileOptions): string[] =>
     }, []);
 export const dbtCompile = async (options: DbtCompileOptions) => {
     try {
-        const args = optionsToArgs(options);
+        const args = optionsToArgs({ ...options });
+        // args.push(` --target-path ${options.projectDir}/target `)
```
[the remark this answers is not in the document]
Comment (rephus) [reply packages/cli/src/handlers/dbt/compile.ts:58]: no, I'll remove it

**Pull request merged**